### ISSUE TITLE
Webpack can't resolve imported css files in custom pages.

### ISSUE THREAD

**Issue by LiamAttClarke** [keystonejs/keystone#2867]
# Bug report

## Describe the bug

Importing a css file in a custom page no longer works with the latest release (`@keystonejs/app-admin-ui v5.12.0`).

## To Reproduce

1. Create a [custom page](https://www.keystonejs.com/keystonejs/app-admin-ui/#pages).
2. Import a local css file.
Example:
```jsx
import './CustomPage.css';

export default () => <div className="some-class"></div>;
```
3. Start Keystone and open in the browser.

## Expected behaviour

The css file should be resolved by Webpack as it did in previous versions of `app-admin-ui`.

## Screenshots

![Annotation 2020-04-30 172950](https://user-images.githubusercontent.com/9361948/80762282-37c7db80-8b0a-11ea-9fd6-40522de696a0.jpg)

## System information

- OS: Windows

## Additional context

@Nikodermus Suggested this might be caused by his recent change (https://github.com/keystonejs/keystone/pull/2860). Perhaps this implicitly excludes local css file resolution?

**Comment by gautamsi** [comment]
this should be reverted and investigated on why the css from `node_modules` not included when there is no `exclude` option specified.

**Comment by MadeByMike** [comment]
Resolved in #2860
